docs/blog/articles/2022-02-11-KServe-0.8-release.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Announcing: KServe v0.8"
+date: 2022-02-11
+description: " KServe v0.8.0 Release Blog Post"
+type: "blog"
+---
+
+### Authors
+ **Dan Sun**, **Paul Van Eck**, **Vedant Padwal**, **Andrews Arokiam** on behalf of the KServe Working Group.
+
+### **Announcing: KServe v0.8**
```

Comment:
Do not actually see the release date on the page view, so let's add it here?

Reply:
Yea, added. Not sure if there is an MKdocs way to automatically add the date somewhere based on whats in the markdown metadata at the very top